API-1729: Update dedicated client documentation to drop EE support (#558)
akeneo/pim-api-docs: content/php-client/getting-started.md

Run the following command to require the libraries in your project:

```bash
$ php composer.phar require akeneo/api-php-client php-http/guzzle6-adapter:^2.0 http-interop/http-factory-guzzle:^1.0
```

::: info
If you don't know which implementation to choose, we strongly recommend you to use Guzzle v6, as in the previous example.
:::

### Enterprise Edition

::: warning

> Enterprise Edition and Community Edition of the client have been fused to a one and unique package.

Reviewer: ```suggestion
Enterprise Edition and Community Edition of the client have been merged into a one and unique package.
```
